### PR TITLE
feat - list component

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created lists component 
Types of lists included:
 - non-bulleted list
 - bulleted list 
 - inline list
 - stacked list
 - stacked list with icon
 - stacked list with avatar
 - stacked list with two lines

list2

### CODE REVIEW

File: css/components/list.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  padding-left: 1.5rem;
+}
+
+.list-group .list-item {
+  padding: 0.5rem 0;
+}
+
+.list-group-inline .list-item {
+  display: inline-block;
+  padding: 0.5rem;
```
Comment (hinanshisuthar) [inline css/components/list.css:18]: You px unit if the final value is <=10 px

Comment (hinanshisuthar) [inline css/components/list.css]: If px unit if the final value is <=10 px

File: css/components/list.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+.list-group-bulleted ul {
+  list-style-type: circle;
+}
+.list-group-bulleted ul ul {
+  list-style-type: square;
+}
+
+.list-group-bulleted .list-item {
+  padding: 0.25rem 0;
```
Comment (hinanshisuthar) [inline css/components/list.css]: Here,  too

**Review by hinanshisuthar — COMMENTED** [review]
Everything looks good to me``

**Review by stutisk — COMMENTED** [review]
Everything looks good to me

**Comment by saadahmadhere** [comment]
Looks good.

File: css/components/list.css (hunk 2 above)
Comment (prachi-sahani) [reply css/components/list.css]: done!

File: css/components/list.css (hunk 1 above)
Comment (prachi-sahani) [reply css/components/list.css]: Done!

**Pull request merged**